DOCS-#3953: Add docs and notebook examples on running Modin with OmniSci (#4001)
modin-project/modin: examples/tutorial/jupyter/README.md

**Note:** Sometimes pip is installing every version of a package. If you encounter that issue,
please install every package listed in `requirements.txt` file individually with `pip install <package>`.

To get required dependencies for `OmnisciOnNative` Jupyter Notebooks
you should create a development environment with `conda`

> using `jupyter_omnisci_env.yml` file located in the current directory:

Author: ```suggestion
using `jupyter_omnisci_env.yml` file located in the respective directory:
```

```bash
conda config --set channel_priority strict

> conda env create -f jupyter_omnisci_env.yml

Author: ```suggestion
conda env create -f execution/omnisci_on_native/jupyter_omnisci_env.yml
```

```

After the environment is created it needs to be activated:

```bash
conda activate jupyter_modin_on_omnisci
```

> **Note:** Notebook for `OmnisciOnNative` working only on Linux.

Author: ```suggestion
**Note:** `Omnisci` engine is available on Linux only for now.
```
